conductor/lib/maya_utils.py

```diff
@@ -103,6 +103,22 @@ def get_plugin_versions():
     return plugin_versions
 
 
+def save_current_maya_scene():
+    '''
+    Saves current Maya scene using standard save command
+    '''
+    logger.debug("Saving Maya scene...")
+    cmds.SaveScene()
+
+
+def get_maya_save_state():
+    '''
+    Returns True if the current Maya scene has unsaved changes
+    '''
+    save_state = cmds.file(q=True, modified=True)
```

Comment:
Nit: simplify to `return cmds.file(q=True, modified=True)`

Reply:
jeez, you read my mind again!  Didn't realize you can simply preface your comment with Nit, and  declare it as the "right" way!